tests/OutputFormatter/JUnitOutputFormatterTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace Tests\SensioLabs\Deptrac\OutputFormatter;
+
+use PHPUnit\Framework\TestCase;
+use SensioLabs\AstRunner\AstMap;
+use SensioLabs\AstRunner\AstMap\AstInherit;
+use SensioLabs\AstRunner\AstMap\FlattenAstInherit;
+use SensioLabs\Deptrac\ClassNameLayerResolverInterface;
+use SensioLabs\Deptrac\Dependency\Result;
+use SensioLabs\Deptrac\DependencyContext;
+use SensioLabs\Deptrac\DependencyResult\Dependency;
+use SensioLabs\Deptrac\DependencyResult\InheritDependency;
+use SensioLabs\Deptrac\OutputFormatter\ConsoleOutputFormatter;
+use SensioLabs\Deptrac\OutputFormatter\JUnitOutputFormatter;
+use SensioLabs\Deptrac\OutputFormatter\OutputFormatterInput;
+use SensioLabs\Deptrac\RulesetEngine\RulesetViolation;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+/**
+ * @author Jan Schädlich <janschaedlich@sensiolabs.de>
```

Comment:
useless "code"

Reply:
So an author tag is generally not needed? @xabbuh wrote it should have one: https://github.com/sensiolabs-de/deptrac/pull/167/files/98ce07cd86eaa99238303828812d5b6e972c631e#diff-571a84413ead5d19e843d892a9f55ebf

What is the general guideline for this project? No comments at all?

Reply:
there is no guideline, but also no technical reason why to embed a `@author` annotation. feel free to discuss this with the sensio team. from my point of view, huge 👎 for any kind of "noise".
`@author` statements also harm refactoring, because nobody likes to remove them.